core/src/main/java/com/netease/arctic/catalog/MixedTables.java

```diff
@@ -137,17 +136,19 @@ protected UnkeyedTable loadUnKeyedTable(TableMeta tableMeta) {
         fileIO, tableMetaStore.getConfiguration()), fileIO, amsClient, catalogMeta.getCatalogProperties());
   }
 
-  public ArcticTable createTableByMeta(TableMeta tableMeta, Schema schema, PrimaryKeySpec primaryKeySpec,
-                                       PartitionSpec partitionSpec) {
+  public ArcticTable createTableByMeta(
```

Comment:
Can you revert these changes to the code format? The old codes seemed good here.